src/hooks/useAddressRandomSeed.ts

```diff
@@ -0,0 +1,18 @@
+import { getLinkedIdentityQuery } from '@/services/datahub/identity/query'
+import { getAccountDataQuery } from '@/services/subsocial/evmAddresses'
+
+export default function useAddressRandomSeed(
```

Comment:
Why is it Random Seed?

Reply:
Renamed to useAddressIdentityId
so its random seed because its used as seed for the random name and random color